### ISSUE TITLE
Passing through natural rifts crashes game

### ISSUE THREAD

**Issue by Asrizuma** [DimensionalDevelopment/DimDoors#80]
I found a rift between stone bricks on top of a battle tower. An entity passing through gave me this every time until I managed to put a door down.
(Additionally, don't know if I'm doing it wrong, but wood and iron dimensional doors seem to be a bit wonky when placing on top of rifts, and rift particles stay even after the remover works)

[crash-2018-07-07_13.45.02-server.txt](https://github.com/DimensionalDevelopment/DimDoors/files/2172954/crash-2018-07-07_13.45.02-server.txt)

**Comment by Asrizuma** [comment]
Don't know if this is important, but the door also stopped the tower from generating past its y-value.

**Comment by Robijnvogel** [comment]
Yeah, we should really check for other worldgen structures before generating ours one of these days.
(Wonky how? Particles die out very slowly, but they should disappear eventually.)

**Comment by Asrizuma** [comment]
The issue isn't the worldgen. It's that entities passing through a natural rift without a door crashes the game. This can result in a world being lost due to an entity always being inside a frame.
As for the wonky bit, the wood door didn't work placing from either side of the rift. The iron door worked one out of four times on a specific side.
Particles die out, but end up frozen in midair and not disappearing, even when the rift has closed.

**Comment by Robijnvogel** [comment]
The TileEntity of that block is not what it is supposed to be as a result of a worldgen conflict. That is why it is crashing.

**Comment by Asrizuma** [comment]
Yeah, all of these might be good things to sort out in the near future.
